src/common/config/env.validation.ts

```diff
@@ -68,19 +76,19 @@ export class EnvironmentVariables {
 
   @IsOptional()
   @IsInt()
-  @Min(1)
+  @IsPositive()
```

Comment:
Do we need IsInt than ?

Reply:
Yes, I think we need `@IsPositive()` and `@IsInt()` together, because `@IsPostitive()` without `@IsInt()` will accept positive real numbers too. I think we want to accept only integers here.